docs/install/index.rst

```diff
@@ -37,17 +41,54 @@ To check your Gammapy installation, you can use this command:
 
     gammapy info
 
-Congratulations! You are all set to start using Gammapy!
 
-If you're new to Python, ipython and Jupyter, read the :ref:`getting-started`
-guide. To learn how to use Gammapy, go to :ref:`tutorials`.
+Using other package managers
+----------------------------
+
+If you don't want to use Anaconda, you can use other package managers. To do so,the following pages contain detailed
+information about Gammapy dependencies and propose various installation options:
+
+* See gammapy install dependencies :ref:`install-dependencies`
+* If you want to install gammapy with pip see :ref:`install-pip`
+* If you want to use other package manager see :ref:`install-other`
 
-The following pages contain detailed information about Gammapy dependencies in
-various installation options:
 
-.. toctree::
-    :maxdepth: 1
+Download tutorials
+------------------
+
+You can now proceed to download the Gammapy tutorial notebooks and the example
+datasets used there (at the moment from CTA, H.E.S.S. and Fermi-LAT). The total
+size to download is about 100 MB.
```

Comment:
The total size of the datasets folder when uncompressed is actually about 180 MB.

Reply:
Changed. Thanks.